packages/playwright/types/test.d.ts

```diff
@@ -4703,7 +4703,7 @@ export interface TestType<TestArgs extends KeyValue, WorkerArgs extends KeyValue
    * @param body Step body.
    * @param options
    */
-  step<T>(title: string, body: () => T | Promise<T>, options?: { box?: boolean }): Promise<T>;
+  step<T>(title: string, body: () => T | Promise<T>, options?: { box?: boolean, location?: Location }): Promise<T>;
```

Comment:
Please note that `Location` here corresponds to the global `lib-dom` `Location`, as opposed to the Playwright's [`Location`](https://github.com/microsoft/playwright/blob/9fc195bff5cd4a5dcb4337d8c9a4400683c9c62b/packages/playwright/types/testReporter.d.ts#L325-L340) interface.

I'd suggest moving the definition of Playwright's `Location` from `testReporter.d.ts` to `playwright/types/test.d.ts` since it's now part of the public API.

@dgozman - what's your view?

Reply:
Looking at it along with the issues you mentioned, I think it's right to change like you said.
The maintainers fixed the bug according to your issue. Thank you for the review. 👍🏻👍🏻